README.md

```diff
@@ -273,6 +273,17 @@ $ export BINDGEN_EXTRA_CLANG_ARGS="-I$(pwd)/frida-gum"
 $ export LIBRARY_PATH="$(pwd)/frida-gum"
 ```
 
+(you can also run these commands via `make`)
```

Comment:
Can you add some detail of which make target to use here?

Reply:
The default make target the first one defined in the file, but I can add it here explicitly if you want?